simphony/cuds/abc_lattice.py

```diff
@@ -8,17 +8,14 @@ class ABCLattice(object):
     ----------
     name : str
         name of lattice
-    type : str
-        type of lattice
-    base_vect : float[3]
-        base vector of lattice
+    prim_cell : PrimitiveCell
```

Comment:
This is just a suggestion that can be ignored, but I prefer the attribute name `primitive_cell` instead of `prim_cell`